types/d3-color/index.d.ts

```diff
@@ -91,6 +105,16 @@ export interface LabColorFactory extends Function {
     readonly prototype: LabColor;
 }
 
+/**
+ * Constructs a new Lab color with the specified l value and a = b = 0.
+ */
+export type GrayColorFactory = (l: number, opacity?: number) => LabColor;
+
+// Using the interface creates an error
+// export interface GrayColorFactory extends Function {
```

Comment:
This comment should be removed. If you still want to mark this as an interface, note that it is not necessary to extend `Function`.